openedx/core/djangoapps/bookmarks/tests/test_models.py

```diff
@@ -55,12 +55,7 @@ def setup_test_data(self, store_type=ModuleStoreEnum.Type.mongo):
             self.course = CourseFactory.create(display_name='An Introduction to API Testing')
             self.course_id = unicode(self.course.id)
 
-            if store_type == ModuleStoreEnum.Type.mongo:
-                bulk_operations_manager = self.store.bulk_operations
-            else:
-                bulk_operations_manager = noop_contextmanager
-
-            with bulk_operations_manager(self.course.id):
+            with self.store.bulk_operations(self.course.id):
```

Comment:
The bug which required us to do this has been fixed so we can use the `bulk_operations_manager` in both cases. This speeds up things quite a bit.